### ISSUE TITLE
Bath is in Somerset

### ISSUE THREAD

**Issue by nigelhorne** [whosonfirst-data/whosonfirst-data#1192]
In wof-venue-gb-20171230T013139-bundle/data/924/837/367/924837367.geojson,  properties->sg:province should be 'Somerset'.

**Comment by nigelhorne** [comment]
I don't know, I'm only looking at the JSON file.

**Comment by nvkelso** [comment]
Here's the whole record in the Spelunker:

- https://spelunker.whosonfirst.org/id/924837367/

Note the `wof:hierarchy`'s `region` set to `85683823` which is "Bath and North East Somerset".

![image](https://user-images.githubusercontent.com/853051/39670255-3bbe526e-50b5-11e8-87f1-addf84847509.png)

Given that I think this issue should be closed. 

Or if the region &/or locality should be something different a new issue created in the main `whosonfirst-data` admin repo to get better polygons.

**Comment by stepps00** [comment]
The `sg:` property mentioned in the original issue comment were imported directly from SimpleGeo; we always maintain the source [property](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties) values, so no change will be made to that property.

We've recently updated administrative records in the UK via #1368 and #1436 and will eventually update venue records' `wof:hierarchy` properties using the new administrative data.

Closing issue as we won't be updating `sg:` properties.
